doc/manpage-newsboat.asciidoc

```diff
@@ -38,6 +38,12 @@ terminals on Unix or Unix-like systems such as GNU/Linux, BSD or Mac OS X.
         _delete-read-articles-on-quit_, _keep-articles-days_, and _max-items_
         settings.
 
+--cleanup::
+        Remove unreferenced entries from the cache. Feeds and their articles
+        will be removed if the feedurl is no longer in the _urls_ file. If 
```

Comment:
When I first read this, I got confused a bit: why would we delete *read* articles if we already deleted *all* of them? I think the reason for the confusion is that there's two separate thoughts in this (single) paragraph.

My advice is to splitt the last sentence into a paragraph of its own, and make it say "*all* read articles will also be deleted", to further emphasize that *this* sentence applies to *all* feeds, not just the ones that are gone from the urls file.

Reply:
Good idea, done: https://github.com/newsboat/newsboat/pull/1250/commits/f03bcc83298e98b2c802b6e8770ca2aefa13f53f